frontend/src/containers/Subscription/SubscriptionFeed.js

```diff
@@ -28,102 +28,112 @@ class SubscriptionFeed extends Component {
     componentDidMount() {
         this.props.onGetSubscriptions()
             .then(() => {
-                this.setState({
-                    subscriptions: this.props.subscriptionItems,
-                });
+                this.setState((prevState) => ({
+                    subscriptions: prevState.subscriptions.concat(this.props.subscriptionItems),
+                    recoCount: 30 - this.props.subscriptionItems.length,
```

Comment:
Could you leave comment about 30 and its purpose?

Reply:
I wanted to maintain the number of feed per 1 page as 30.

Reply:
I think it would be helpful if you leave the comment about it in codes. And I think it's related with https://github.com/swsnu/swpp2019-team3/pull/182#discussion_r355106410.

Reply:
Good I will change backend code and leave some comment.